ruff.toml

```diff
@@ -14,12 +14,16 @@ extend-select = [
 	"ANN2", # missing-return-type-*
 	"FA", # flake8-future-annotations
 	"F404", # late-future-import
+	"PT", # flake8-pytest-style
 	"PYI", # flake8-pyi
 	"UP", # pyupgrade
 	"TRY",
 	"YTT", # flake8-2020
 ]
 ignore = [
+	"PT007", # temporarily disabled, TODO: configure and standardize to preference
+	"PT011", # temporarily disabled, TODO: tighten expected error 
+	"PT012", # pytest-raises-with-multiple-statements, avoid extra dummy methods for a few lines, sometimes we explicitly assert in case of no error
```

Comment:
I felt like the changes this was asking for didn't fit the style of a handful of current tests, would've added more boilerplate without helping that much prevent false-positives.